fenix/app/src/main/java/org/mozilla/fenix/onboarding/view/JunoOnboardingScreen.kt

```diff
@@ -199,7 +199,7 @@ private class DisableForwardSwipeNestedScrollConnection(
             if (pagerState.currentPageOffset < 0) {
                 return Offset.Zero
             } else {
-                Offset(available.x, available.y)
+                Offset(available.x, 0f)
```

Comment:
Small consistency nitpick: should this and line 193 be `return` statements like line 200?